Project/Assets/Scripts/Player/PlayerAttack.cs

```diff
@@ -69,30 +72,38 @@ public class PlayerAttack : MonoBehaviour {
             meleeAttackTrigger.enabled = false;
         }
     }
+    #endregion
 
-    private void CastAbility(Abilities ability)
+    //Casts players ability
+    private void CastAbility(AbilityTest ability)
     {
-        if (ability.isOffCooldown)
+        //Check to see ability is off cooldown
+        if (ability.offCooldown && (player.playerHUD.energySlider.value - ability.energy) > 0)
         {
-            anim.Play(ability.InputTag, 0);
+            //play the correct animation
+            anim.Play(ability.animationTag, 0);
+            //set the correct trigger
             meleeAttackTrigger.enabled = true;
-            meleeAttackTrigger.SendMessage("updateDamage", ability.Damage);
```

Comment:
Would we want to centralize all the static types?,(have a single static class that contains these Static types?) or just have the associated static type be in the associated class (have a static type in MeleeAttackTrigger script with the string value) ?